src/App.js

```diff
@@ -1,27 +1,27 @@
 import React from 'react';
-import { db } from './lib/firebase';
-import { useCollection } from 'react-firebase-hooks/firestore';
+// import { db } from './lib/firebase';
+// import { useCollection } from 'react-firebase-hooks/firestore';
```

Comment:
Documentation in code is good, and code that is self documenting through clear variables is great. For the same reasons, commented code can make code more difficult to handle cognitively. We should delete the commented out code throughout the PR, which should be pretty easy to do! If you have code you don't want to lose for future use, saving it locally or using a [Github Gist](https://gist.github.com/) are good choices!